src/main/io/osd.c

```diff
@@ -1579,6 +1584,12 @@ static void osdShowStats(uint16_t endBatteryVoltage)
         tfp_sprintf(buff, "%d%c", osdGetMetersToSelectedUnit(stats.max_distance), osdGetMetersToSelectedUnitSymbol());
         osdDisplayStatisticLabel(top++, "MAX DISTANCE", buff);
     }
+
+    if (osdStatGetState(OSD_STAT_TOTAL_DISTANCE) && featureIsEnabled(FEATURE_GPS)) {
```

Comment:
If we are going this way, then we should also combine the `featureIsEnabled(FEATURE_GPS)` condition.